iroh/examples/rpc.rs

```diff
@@ -21,10 +20,7 @@ pub fn setup_logging() {
         .ok();
 }
 
-async fn run<S>(builder: iroh::node::Builder<S>) -> anyhow::Result<()>
-where
-    S: Store,
-{
+async fn run(builder: iroh::node::Builder) -> anyhow::Result<()> {
     let node = builder
         .enable_rpc()
```

Comment:
I was a bit confused at first what node does now. I thought the idea was that iroh is a pure reexport crate and whenever you build a node like thing you DIY it using the router API. E.g. this enable_rpc enables rpc, but only for the node api, which is almost never what people want if they want rpc.

Is the plan to change this at some point?

Reply:
yes, that `Node` will go away, as a follow up to this PR